src/statsmodels/contrasts.jl

```diff
@@ -292,7 +292,6 @@ function contrasts_matrix(C::HelmertCoding, baseind, n)
     mat = mat[[baseind; 1:(baseind-1); (baseind+1):end], :]
     return mat
 end
-    
```

Comment:
Removes trailing whitespace. Note that the diff shows that this line had the whitespace removed in favor of an empty line below.